react/features/whiteboard/middleware.ts

```diff
@@ -0,0 +1,222 @@
+/* eslint-disable import/order */
+/* eslint-disable require-jsdoc */
+
+import { generateCollaborationLinkData } from '@jitsi/excalidraw';
+
+// @ts-ignore
+import { IStore } from '../app/types';
+
+// @ts-ignore
+import { setTileView } from '../video-layout';
+import {
+    getLocalParticipant,
+    PARTICIPANT_LEFT,
+    participantJoined,
+    participantLeft
+
+    // @ts-ignore
+} from '../base/participants';
+
+// @ts-ignore
+import { getCurrentConference } from '../base/conference/functions';
+import { CONFERENCE_JOIN_IN_PROGRESS } from '../base/conference/actionTypes';
+
+// @ts-ignore
+import { MiddlewareRegistry, StateListenerRegistry } from '../base/redux';
+
+import { TOGGLE_WHITEBOARD } from './actionTypes';
+import { getCollabLink, getWhiteboardId, isWhiteboardOpen } from './functions';
+import { USERNAME_UPDATE_STATUS, WHITEBOARD, WHITEBOARD_PARTICIPANT_NAME, WHITEBOARD_STORAGE_KEY } from './constants';
+import { disableWhiteboard, enableWhiteboard, setUsernameStatus } from './actions';
+import { PARTICIPANT_UPDATED } from '../base/participants/actionTypes';
+
+// @ts-ignore
+import { getCurrentRoomId } from '../breakout-rooms/functions';
+
+// @ts-ignore
+import { addStageParticipant } from '../filmstrip/actions.web';
+
+declare let APP: any;
+
+
+/**
+ * Sends the WHITEBOARD command.
+ *
+ * @returns {void}
+ */
+// @ts-ignore
+const sendWhiteboardCommand = ({
+    conference,
+    isEnabled,
+    roomId,
+    roomKey,
+    participantId,
+    whiteboardId
+}: {
+    conference: any;
+    isEnabled: boolean;
+    participantId?: string;
+    roomId?: string;
+    roomKey?: string;
+    whiteboardId: string;
+}): void => {
+    conference.sendCommandOnce(WHITEBOARD, {
```

Comment:
If you send it only once then if you leave it won't be available for future participants who join.